### ISSUE TITLE
[Feature Request] Adding manual or better JEI integration (or both)

### ISSUE THREAD

**Issue by davqvist** [cyanobacterium/ElectricAdvantage#16]
Hi I tried your mod out and it seems pretty neat, considering to include it in my next pack, but I couldn't figure out what a lot of the machines do due to the lack of a guide, tutorial or at least JEI integration for things other than normal crafting recipes.

I think a guide book would be very helpful, but not 100% needed (a question mark in the corner of the machines where you can hover over popping up a short explanation would be sufficient for example), but I definitely thing that JEI integration is crucial.

I might be stupid, but I couldn't find out at all how the Automated Assembler or the Growth Chamber works. How to get Petro-Plastic Ingots using the Petro-Plastic Refinery I could only find out after trial and error and only due to the fact that both share the name. For the Distiller it was the same, I just tried several fluids until one worked.

To summarize, to realistically include it in any modpack, I'm pretty sure a better explanation through more infos and JEI integration is needed. Keep up the good work.

**Comment by jriwanek** [comment]
https://github.com/minecraftmoddevelopmentmods/electricadvantage/issues

This is the old tracker, but we will look into the issue

**Comment by davqvist** [comment]
Oh, didn't realize, my bad. Thanks for answering.
